Backend/Tournament_Admin/TournamentAdminService/src/main/java/TournamentAdminService/config/AppConfig.java

```diff
@@ -0,0 +1,39 @@
+package TournamentAdminService.config;
+
+import org.hibernate.boot.model.naming.PhysicalNamingStrategy;
+import org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+
+/**
+ * Application configuration class for defining common beans.
```

Comment:
documentation (is this the default one)

Reply:
oh no i took this from ngee tian cuz i was facing naming convention issue